src/utilities/effects.ts

```diff
@@ -178,13 +178,13 @@ const effects = {
     try {
       checkConstraintsStatus.set(Status.Incomplete);
       const { id: planId } = get(plan);
-      const data = await reqHasura<{ violationsMap: ConstraintViolationsResponseMap }>(gql.CHECK_CONSTRAINTS, {
+      const data = await reqHasura<{ violationsMap: ConstraintViolationsResponse }>(gql.CHECK_CONSTRAINTS, {
```

Comment:
Instead of `violationsMap` can we change the `CHECK_CONSTRAINTS` to return `violations` instead of `violationsMap`?